src/tableExplorer/tableExplorerWebViewController.ts

```diff
@@ -359,7 +360,7 @@ export class TableExplorerWebViewController extends ReactWebviewPanelController<
                     rowsLoaded: subsetResult.rowCount.toString(),
                 });
             } catch (error) {
-                state.isLoading = false;
+                state.loadStatus = ApiStatus.Error;
```

Comment:
How does the error state display in the UI?  Probably should have the message displayed so the user knows to refresh it (better than just showing an error toast)